bin/src/model/dto/config/package.dart

```diff
@@ -0,0 +1,34 @@
+class CachedPackage {
+  final String name;
+  final String rootUri;
+  final String packageUri;
+
+  CachedPackage({
+    required this.name,
+    required this.rootUri,
+    required this.packageUri,
+  });
+
+  factory CachedPackage.fromJson(Map<String, dynamic> json) {
+    var rootUri = json['rootUri'] as String;
```

Comment:
A lot of conversion in the fromJson. Is this an outside source? If not, why is this data not sanitized before?

Reply:
This data is coming from `.dart_tool/package_config.json` which is generated by flutter packages get. The way it is currently setup does not work for us. So I do some conversions to have a consistent way of working with these files.